virtool/ws/server.py

```diff
@@ -78,6 +80,37 @@ def remove_connection(self, connection: WSConnection):
         except ValueError:
             pass
 
+    async def handle_redis_ws_messages(self):
+        channel = self._redis.subscribe("close_websocket_connections")
+
+        while await channel.wait_message():
+            session_id = await channel.get(encoding="utf-8")
+            await self.close_connections_with_session_id(session_id)
```

Comment:
Remove the new Redis logic, we will assume that either the client will disconnect itself, or that the scan will do it soon enough for now.